core/src/test/scala/tech/beshu/ror/unit/boot/ReadonlyRestStartingTests.scala

```diff
@@ -1354,16 +1352,16 @@ class ReadonlyRestStartingTests
         mapWithIntervalFrom(refreshInterval) ++
           mapWithMaxYamlSize(maxYamlSize) ++
           Map(
-            "com.readonlyrest.settings.loading.delay" -> "0"
+            "com.readonlyrest.settings.loading.delay" -> "1"
```

Comment:
is it intentional?

Reply:
It's intentional. The FiniteDuration type has been changed to the PositiveFiniteDuration. The value of 0 became invalid, so the default was used (5 seconds) in the tests. With this change, I reduced all unit test execution time by 5 minutes.

Reply:
but with this little change, we change the interface too. Because some may have this configured to 0. Is it worth it?

Reply:
The type was changed a few months ago, I only changed the value used in tests to shorten test execution.